workers/common/src/main/kotlin/common/env/ConfigFileBuilder.kt

```diff
@@ -47,6 +47,9 @@ class ConfigFileBuilder(val context: WorkerContext) {
     companion object {
         private val logger = LoggerFactory.getLogger(ConfigFileBuilder::class.java)
 
+        /** The names of environment variables containing the proxy configuration. */
+        private val PROXY_VARIABLES = listOf("HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY")
```

Comment:
IMO a solution based on enum would be better: replace this with an enum,
add a `getEnv` function to the enum, build a EnumMap of enum to their env value and replace array accesses underneath with map lookup.
The problem is, if somebody change this array without adapting `printProxySettings`, it will bug.
Also, using enum is better for finding usages in the IDE.
What do you think ?

Reply:
Good idea. I have changed the implementation accordingly.